RCodes/NonBinaryModelCheck.R

```diff
@@ -0,0 +1,164 @@
+library(data.table)
+library(glmnet)
+library(ggplot2)
+library(tidyverse)
+library(caret)
+library("ordinalNet")
+library("ordinalForest")
+library(tidyverse)
+library(caret)
+library("datarium")
+library(ggplot2)
+library("Metrics")
+library("MLmetrics")
+
+LogLossBinary = function(actual, predicted, eps = 1e-15) {
+  predicted = pmin(pmax(predicted, eps), 1-eps) - (sum(actual * log(predicted) + (1 - actual) * log(1 - predicted))) / length(actual)
+}
+y_test <- as.numeric(y_test)
+ordnet.pred 
+LogLossBinary(y_test,ordnet.pred)
+
+combined.binary.input.data<- data.table::fread("/Users/akhilachowdarykolla/Documents/Coding/development/PredictiveModelling/coaching_cwis_aggregrate_districts.csv")
+head(combined.binary.input.data)
+fold.vec <- sample(rep(1:10,l=nrow(combined.binary.input.data)))
+combined.binary.input.data <- data.table::data.table(combined.binary.input.data,fold=factor(fold.vec))
+
+combined.binary.input.data$BinaryETL <- ""
+for(i in 1:nrow(combined.binary.input.data)) {
+  row <- combined.binary.input.data[i,]
+  etlScore <- row$ETL.AVERAGE
+  print(etlScore)
+  if (etlScore >= 0 && etlScore <= 2.8) {
+    combined.binary.input.data[i,]$BinaryETL <- 1
+  } else{
+    combined.binary.input.data[i,]$BinaryETL <- 2
+  }
+}
+write.csv(combined.binary.input.data,"/Users/akhilachowdarykolla/Documents/Coding/development/PredictiveModelling/onetwo_factored_combined_binary.input_data.csv", row.names = FALSE)
+
+
+binary.input.data<- data.table::fread("/Users/akhilachowdarykolla/Documents/Coding/development/PredictiveModelling/onetwo_factored_combined_binary.input_data.csv")
+head(binary.input.data)
+binary.input.data <- binary.input.data[,-c(1,2,19)]
+
+head(binary.input.data)
+head(binary.input.data$BinaryETL,100)
+unique(binary.input.data$BinaryETL)
+
+binary.input.data$BinaryETL <- as.factor(binary.input.data$BinaryETL)
+
+
+binary.input.data <- as.data.frame(binary.input.data)
+
+for(j in 1:98){
+  binary.input.data[,j][is.na(binary.input.data[,j])] <- 0
+}
+
+for(k in 1:98){
+  binary.input.data[,k] <- as.numeric(binary.input.data[,k])
+}
```

Comment:
what are j and k? please avoid single letter variable names, use informative names instead (col.j, col.k would be better since that would tell reader it is a column index)
also you can probably remove the for loop